boards/esp32-s3-wroom-1-n16r8.json

```diff
@@ -0,0 +1,50 @@
+{
+    "build": {
+        "arduino": {
+            "ldscript": "esp32s3_out.ld",
+            "memory_type": "qio_opi",
+            "partitions": "default_16MB.csv"
+        },
+        "core": "esp32",
+        "extra_flags": [
+            "-DBOARD_HAS_PSRAM",
+            "-DARDUINO_ESP32_S3_N16R8",
+            "-DARDUINO_USB_MODE=1",
+            "-DARDUINO_USB_CDC_ON_BOOT=1",
```

Comment:
`-DARDUINO_USB_CDC_ON_BOOT=1` probably shouldn't be enabled at this level, but in `platform.io` if it's needed.

I'm using this chip on an [ESP32-S3-DevKitC-1](https://docs.espressif.com/projects/esp-idf/en/stable/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1.html). If I include this flag is included then I have to hold down the 'Boot' button and press 'Reset' _every time_ I want to reflash. If this flag is removed then I can just 'Upload and Monitor' as normal.

Reply:
> `-DARDUINO_USB_CDC_ON_BOOT=1` probably shouldn't be enabled at this level, but in platform.io if it's needed.

Same for `-DARDUINO_USB_MODE=1` !